composer.json

```diff
@@ -89,6 +91,9 @@
         "symfony/browser-kit": "^6.2",
         "symfony/http-client": "^6.2"
     },
+    "provide": {
+        "ext-sodium": "*"
+    },
```

Comment:
I added this requirement to be able to run tests. If https://github.com/paragonie/sodium_compat/pull/156 is accepted, we will have to remove it.

Reply:
Solution was refused by `paragonie/sodium_compat` maintainers because their polyfill is not covering 100% of the extension.
We have to keep this workaround.